frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -299,8 +299,21 @@ export const experimentLogic = kea<experimentLogicType>([
     }),
     listeners(({ values, actions }) => ({
         createExperiment: async ({ draft, runningTime, sampleSize }) => {
+            const { experiment, experimentInsightType } = values
             let response: Experiment | null = null
             const isUpdate = !!values.experimentId && values.experimentId !== 'new'
+
+            if (
+                experimentInsightType === InsightType.FUNNELS &&
+                (!experiment.filters.events?.length || experiment.filters.events.length < 2)
```

Comment:
wouldn't work correctly if the second/first step is an action 😅

Reply:
🙈 fixed!